Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_XCUITests.xcscheme

```diff
@@ -168,8 +168,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Fennec"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
-      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
```

Comment:
Is there a bug on XCode or the iOS simulator instead?

Reply:
I think it's an XCode bug, it doesn't seem to like having the debugger attached on devices with Apple silicon. This is something that normally shouldn't cause any problems. I don't think disabling it should cause any issues tho. Let me know if you do see anything unusual while writing new tests tho.

Reply:
Could this change ok on Intel as well?

Reply:
Yup, it shouldn't make a difference on Intel.